zone/iamalsoandy

```diff
@@ -0,0 +1,2 @@
+iamalsoandy IN A 81.174.166.128 
+www.iamalsoandy IN CNAME iamalsoandy.o
```

Comment:
```suggestion
www.iamalsoandy IN CNAME iamalsoandy.o.
```

If you include the `.o` suffix you need a trailing dot to indicate it's an absolute domain. Otherwise LGTM.

Reply:
Great - thanks! 

Pardon my checking of this but should I now change my repo and submit another pull request?